test/src/main/java/io/strimzi/test/k8s/cmdClient/BaseCmdKubeClient.java

```diff
@@ -181,6 +181,8 @@ public K replace(File... files) {
     @SuppressWarnings("unchecked")
     public K applyContent(String yamlContent) {
         try (Context context = defaultContext()) {
+            // TODO remove this
+            LOGGER.info(yamlContent);
```

Comment:
what about this TODO to remove?

Reply:
I am still trying to debug reason why last test failing. I will remove it later. Also I moved this PR to `draft`, because it's not ready net. Sorry for confusion